renv.lock

```diff
@@ -283,7 +283,7 @@
       "RemoteUsername": "ccao-data",
       "RemoteRepo": "ccao",
       "RemoteRef": "master",
-      "RemoteSha": "6bdb2f2cab811c3450d91726a784804bd8bca971",
+      "RemoteSha": "fe992f9bb6e75c97a0c8fa742106096adbabb642",
```

Comment:
This is just updating the CCAO package to the latest version. It's tangential to this PR.